docs/plans/2026-01-20-persistent-storage-design.md

```
@@ -2083,6 +2083,19 @@ git commit -m "feat: add SaveStatus component with bits-ui Progress"
 </style>
 ```
 
+**Note: Future Enhancement - Persistence Integration**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)